float32 north_angle
float32 north_angle_unc
